### DISCUSSION TITLE
Mobile hotspot plus wlan1 seems provides very slow connection

### DISCUSSION THREAD

**Discussion by njerca** [Mygod/VPNHotspot#441] [Q&A]
With the way to share VPN connection by above method, the speed of access is very slow. 

The temporary hotspot way is much better, seems no difference than regular connection. However, the temporary hotspot changes SSID and password everytime when it is turned on.

Is there anything can config to get the speed to normal in wlan mode? Or make the way of temporary hotspot could use predefined config as setup regular hotspot?

Thanks.

**Comment by Mygod** [comment]
Try repeater?

**Reply by njerca** [reply]
Yes, repeater works much better. Thanks. 

Only issue is whenever want to change something of the repeater, the on/off button would be greyed out. Have to reboot device to get it back.

**Reply by Mygod** [reply]
Try toggling airplane mode?
